api/src/pcapi/routes/backoffice_v3/offerers.py

```diff
@@ -331,13 +350,21 @@ def get_managed_venues(offerer_id: int) -> utils.BackofficeResponse:
                 educational_models.CollectiveDmsApplication.state,
                 educational_models.CollectiveDmsApplication.lastChangeDate,
             ),
+            sa.orm.joinedload(offerers_models.Venue.registration).load_only(
+                offerers_models.VenueRegistration.target,
+                offerers_models.VenueRegistration.webPresence,
+            ),
         )
         .all()
     )
 
+    formatted_venues = list(venues)
+    for venue in formatted_venues:
+        venue.formatted_target = _format_target(venue.target)
+        venue.formatted_web_presence = _format_web_presence(venue.web_presence)
```

Comment:
```suggestion
        venue.formatted_target = TARGET_MAPPING.get(target, "") if venue.target else ""
        venue.formatted_web_presence = web_presence.split(", ") if venue.web_presence else ""
```
On gagnerait en simplicité avec ça, à mon humble avis

Reply:
J'aurais plutôt mis un "_formatter_" défini ans les filtres pour rester cohérent avec le reste, et laissé `venues=venues` plus bas.

Reply:
En effet, bien vu